src/snabbdom.ts

```diff
@@ -138,13 +136,13 @@ export function init(modules: Array<Partial<Module>>, domApi?: DOMAPI) {
 
   function invokeDestroyHook(vnode: VNode) {
     let i: any, j: number, data = vnode.data;
-    if (data !== undefined) {
+    if (isDef(vnode.sel)) {
       if (isDef(i = data.hook) && isDef(i = i.destroy)) i(vnode);
       for (i = 0; i < cbs.destroy.length; ++i) cbs.destroy[i](vnode);
       if (vnode.children !== undefined) {
         for (j = 0; j < vnode.children.length; ++j) {
           i = vnode.children[j];
-          if (i != null && typeof i !== "string") {
+          if (typeof i !== "string") {
```

Comment:
Hey @paldepind ,
As you mentioned, checking if child is string.
According to child type, it can only be VNode | string.